### PR TITLE
Edits help text and rearranges placement

### PR DESCRIPTION
**Pull request by annegentle**
- Moves DESCRIPTION above SYNOPSIS in man page
    - Moves config above init for commands
    - Edits USAGE text to emphasize usage
    - Edits DESCRIPTION text to describe tool rather than making of it

Issue #204

### CODE REVIEW

**Comment by kenperkins** [comment]
This commit has incorrect import statements. Please fix this before we proceed.

**Comment by annegentle** [comment]
Ah, I wondered about those. I'll work on a clean copy of the repo.

**Comment by annegentle** [comment]
I can't change the label from wip but this is ready for review [rfr].

File: main.go [hunk 1]
```diff
@@ -37,35 +36,31 @@ func main() {
...
-			Name: "init",
-			Usage: strings.Join([]string{"For Linux and OS X, creates the `rack` man page and sets up command completion for the Bash shell.",
-				"\tFor Windows, creates a `posh_autocomplete.ps1` file in the `$HOME/.rack` directory. You must run the file to set up command completion."}, "\n"),
+			Name:   "configure",
+			Usage:  "Interactively create a config file for Rackspace authentication.",
+			Action: configure,
+		},
+		{
+			Name:  "init",
+			Usage: "[Linux/OS X only] Creates the rack man page and sets up command completion for the Bash shell.",
```
Comment (jrperritt) [inline main.go]: Why was the Windows text removed?

**Comment by jrperritt** [comment]
Ping @annegentle

File: main.go (hunk 1 above)
Comment (annegentle) [inline main.go]: Ah, didn't mean to. What does `strings.Join` do? I'll try again.

Comment (jrperritt) [inline main.go]: I takes a slice of strings and joins them with a specified string. It is used here to break the string up into individual lines and then join them with a new-line character ("\n").

**Comment by jrperritt** [comment]
closing in favor of #282

**Pull request closed without merging**